src/core/Table/Table.tsx

```diff
@@ -215,6 +253,9 @@ export const Table = <
     if (action.type === TableActions.toggleSortBy) {
       onSort?.(newState);
     }
+    if (action.type === TableActions.setFilter) {
```

Comment:
Other PR has here expand action.. should we switch to `switch` statement or something smarter...?